frontend/ui/screensaver.lua

```diff
@@ -161,6 +162,7 @@ function Screensaver:expandSpecial(message, fallback)
     local series = _("N/A")
     local time_left_chapter = _("N/A")
     local time_left_document = _("N/A")
+    local batt_lvl = 0
```

Comment:
N/A is fine ;).

Reply:
Done, but could you enlighten me on these `_("string")` strings? Couldn't find anything about them online

Reply:
> these `_("string")` strings


Translations
https://github.com/koreader/koreader/blob/51ba33ca9009863866452c146167869d81e029c1/frontend/ui/screensaver.lua#L26

Reply:
Thanks, learned a lot. In this case, `batt_lvl` is just an integer though, and the other numbers for the message just above like `currentpage` are not i18n'ed. Why this one?

Reply:
Translations not the issue, the default value on not hasBattery device is ;). In which case, not available makes more sense than zero.

Reply:
I understand now. So if the device has no battery, we want the battery level to show up as N/A, not 0.